cdap-data-fabric/src/test/java/co/cask/cdap/data/hbase/HBaseTestBase.java

```diff
@@ -55,7 +55,16 @@ public String getZkConnectionString() {
 
   // Test startup / teardown
 
-  public abstract void startHBase() throws Exception;
+  public final void startHBase() throws Exception {
+    // Tune down the connection thread pool size
+    getConfiguration().setInt("hbase.hconnection.threads.core", 5);
+    getConfiguration().setInt("hbase.hconnection.threads.max", 10);

```

Comment:
It's through use of the RegionCoprocessorEnvironment.getTable() call.  So this config change will have a big impact in the ConsumerConfigCache usage.

The HTable config I suggested may not make much of a difference since HTable always sets the core pool size to 1.

I'll keep looking for other configs which could help.